### PR TITLE
Set minimum versions in large-image extras

### PR DESCRIPTION
**Pull request by banesullivan**
For #828

### CODE REVIEW

**Comment by banesullivan** [comment]
Ensuring `setuptools-scm` is installed pre-install might be a pain

**Comment by manthey** [comment]
#831 affects this.

We could guard the version production string (e.g.,
```
try:
    from setuptools_scm import get_version

    version = get_version(root='..', local_scheme=prerelease_local_scheme, fallback_version='development')
    limit_version = f'>={version}'
except ImportError:
    version = limit_version = ''
```

and then I think we still want to use `use_scm_version` rather than `version`.

**Comment by manthey** [comment]
I think that in the except

> ```python
> except (ImportError, LookupError):
>     version = limit_version = ''
> ```

we won't need to set `version` since we won't use it later.

**Comment by banesullivan** [comment]
Right - meant to clean that up

I'll push these changes

**Comment by banesullivan** [comment]
Huh, the diff looks off - I may need to squash the commits here

File: girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js [hunk 1]
```diff
@@ -662,6 +665,46 @@ var GeojsImageViewerWidgetExtension = function (viewer) {
...
+                addval = [
+                    addval[0] - addval[3], addval[1] - addval[4],
+                    addval[0] + addval[3], addval[1] - addval[4],
+                    addval[0] + addval[3], addval[1] + addval[4],
+                    addval[0] - addval[3], addval[1] + addval[4]
+                ];
+            }
+            return origval.concat([-1, -1]).concat(addval);
+        },
+
```
[the remark this answers is not in the document]
Comment (manthey) [reply girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js:707]: Those are from a different PR.  Maybe you should have rebased.

[the remark this answers is not in the document]
Comment (banesullivan) [reply girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js]: Rebased

**Comment by banesullivan** [comment]
Rebased and ready for review

**Review by manthey — APPROVED** [review]

**Comment by manthey** [comment]
Thanks @banesullivan .

**Pull request merged**